file-formats/dobj/examples/z_aff_example_dobj.dobj.json

```
@@ -1,21 +1,18 @@
-{
-  "header": {
-    "abapLanguageVersion": "standard"
-  },
-  "attributes": {
-    "destructionClass": "CL_SFLIGHT_DESTRUCTION",
-    "applicationComponent": "BC-ILM-IRM"
-  },
-  "structureDefinition": [
-    {
-      "item": "1",
-      "parentTable": "",
-      "dependentTable": "SPFLI"
-    },
-    {
-      "item": "2",
-      "parentTable": "SPFLI",
-      "dependentTable": "SFLIGHT"
-    }
-  ]
-}
+{
+  "header": {
+    "abapLanguageVersion": "standard"

```

The header should contain a `description` and the `originalLanguage`. However, `"abapLanguageVersion": "standard"` can be removed since this is the default :)

```suggestion
    "description": "Example destruction object",
    "originalLanguage": "en"

```